ProductsAnalysisWeb/Views/App/Home.cshtml

```diff
@@ -89,6 +89,9 @@
     max-height: 300px;
 }
 </style>
+<head>
+    <link rel="stylesheet" href="~/css/site.css" />
```

Comment:
Usually, link to stylesheet looks like this:
href="css/site.css"

It means 'site.css' is stored under folder 'css' in the root of your site...